streaming/index.js

```diff
@@ -249,7 +240,7 @@ if (cluster.isMaster) {
           const targetAccountIds = [unpackedPayload.account.id].concat(unpackedPayload.mentions.map(item => item.id)).concat(unpackedPayload.reblog ? [unpackedPayload.reblog.account.id] : []);
           const accountDomain    = unpackedPayload.account.acct.split('@')[1];
 
-          if (req.filteredLanguages.indexOf(unpackedPayload.language) !== -1) {
+          if (req.filteredLanguages instanceof Array && req.filteredLanguages.indexOf(unpackedPayload.language) !== -1) {
```

Comment:
I think that "`Array.isArray(req.filteredLanguages) && req.filteredLanguages.includes(unpackedPayload.language)`" is better.

- [`Array.isArray` (ES 5.1)](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray)
- [`Array.prototype.includes` (ES 2016)](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Both are implemented in Node.js